drivers/ma_reports/app/models/ma_reports/csg_engage/report_components/household.rb

```diff
@@ -12,7 +12,9 @@ def initialize(hoh_enrollment)
       @hoh_enrollment = hoh_enrollment
     end
 
-    field('Household Identifier') { hoh_enrollment.id.to_s }
+    field('Household Identifier') do
+      [hoh_enrollment.id, hoh_enrollment.project.id, hoh_enrollment.data_source.id].map { |id| id.to_s(16) }.join('::')
```

Comment:
So, in this case using `.id` on the enrollment would be sufficient, but I _think_ you want to tie it to eh `HouseholdID` field in case they change who is HoH.  Using `project.id` also means you don't need `data_source.id`.

We usually do something like this:
```suggestion
      Digest::MD5.hexdigest([hoh_enrollment.HouseholdID, hoh_enrollment.project.id].join('|'))
```

That should give a consistent value based on the project and HouseholdID and should be the correct length to fit in the HouseholdID field.

Reply:
@eanders The field length limit is 20 chars, and I think MD5 hashes are 32. I guess we could just truncate it, but technically that could cause collisions, though unlikely.

Reply:
@eanders This is fixed now